drivers/sensor/bh1750/bh1750.c

```diff
@@ -0,0 +1,145 @@
+/* bh1750.c - Driver for BH1750 ambient light sensor */
+/*
+ * Copyright (c) 2023 Sngular People SL.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT rohm_bh1750
+/* We need to define a device: DEVICE_DT_INST_DEFINE */
```

Comment:
The comments per include are probably a bit verbose

Reply:
I agree on it. This driver was my first approach to the Sensor API and I documented so much. I forgot to delete these verbose comments, I will remove them.